.vortex/cli/tests/Unit/Handlers/NamesHandlerDiscoveryTest.php

```
@@ -33,7 +34,8 @@ public static function dataProviderRunPrompts(): \Iterator {
         OrgMachineName::id() => 'prompted_project_org',
         Domain::id() => 'prompted-project.com',
         ModulePrefix::id() => 'pp',
-        Theme::id() => 'prompted_project',
+        Theme::id() => Theme::CUSTOM,
```

PHPMD (2.15.0)

[error] Avoid using static access to class '\DrevOps\VortexCli\Prompts\Handlers\Theme' in method 'dataProviderRunPrompts'. (undefined)

(StaticAccess)